routes/api/comments/index.js

```diff
@@ -18,6 +18,18 @@ router.get('/', authorization.needed('admin'), (req, res, next) => {
     user_id = null
   } = req.query;
 
+  // everything on this route requires admin privileges besides listing comments for owner of said comments
```

Comment:
Can we add a tests for this?  Right now we have one that returns all the comments when the admin user is injected.